### ISSUE TITLE
[jom] update to 1.1.4

### ISSUE THREAD

**Issue by Voskrese** [microsoft/vcpkg#10200]
jom 1.1.4, the parallel make tool.

_Other information that may be useful (release notes, etc...)_
https://github.com/qt-labs/jom/blob/master/changelog.txt

https://github.com/qt-labs/jom/commits/master

> Commits on Sep 18, 2019
>     **Support paths longer than MAX_PATH** 

deps - Qt5

`
vcpkg env --triplet x64-windows`

`cmake E:\tools\vcpkg\downloads\tools\jom\jom-1.1.4 -G "NMake Makefiles" -DCMAKE_PREFIX_PATH="E:\tools\Qt\Qt5.14.0\5.14.0\msvc2017_64" -DCMAKE_INSTALL_PREFIX=E:\tools\vcpkg\downloads\tools\jom\jom-1.1.4\install -DQT_QMAKE_EXECUTABLE:PATH=E:\tools\Qt\Qt5.14.0\5.14.0\msvc2017_64\bin\qmake.exe -DCMAKE_BUILD_TYPE=release -DCMAKE_CONFIGURATION_TYPES=Release`

`nmake && nmake install
`

> == Environment variables ==
> 
> Like nmake, jom reads default command line arguments from an environment variable: JOMFLAGS.
> If JOMFLAGS is not set, MAKEFLAGS is read.
> This is useful to set up separate flags for nmake and jom, e.g.
>     set MAKEFLAGS=L
>     set JOMFLAGS=Lj8

`jom.exe /?
`

> jom 1.1.4 - empower your cores
> 
> Usage: jom @commandfile
>        jom [options] [/f makefile] [macro definitions] [targets]
> 
> **nmake compatible options:**
> /A build all targets
> /D display build information
> /E override environment variable macros
> /F <filename> use the specified makefile
> /G display included makefiles
> /H show help
> /I ignore all exit codes
> /K keep going - build unrelated targets on error
> /N dry run - just print commands
> /NOLOGO do not print logo
> /P print makefile info
> /R ignore predefined rules and macros
> /S silent mode
> /U print content of inline files
> /L same as /NOLOGO
> /W print the working directory before and after other processing
> /X <filename> write stderr to file.
> /Y disable batch mode inference rules
> 
> **jom only options:**
> /DUMPGRAPH show the generated dependency graph
> /DUMPGRAPHDOT dump dependency graph in dot format
> /J <n> use up to n processes in parallel
> /VERSION print version and exit

**Comment by NancyLi1013** [comment]
Hi @Voskrese 

Thanks for posting this issue.
It seems that there is no released version 1.1.4 for jom now.
Besides this, the url you provided above is not the same as that vcpkg used now.

**Comment by Voskrese** [comment]
Hi @NancyLi1013 
I suggest compiling the utility separately and uploading it here

https://github.com/microsoft/vcpkg/releases

![2020-02-25_232202](https://user-images.githubusercontent.com/10104740/75261755-7f8a6800-5826-11ea-867f-91884578d437.png)

**Comment by Voskrese** [comment]
just in case, may be useful to someone


[jom-1.1.4.zip](https://github.com/microsoft/vcpkg/files/4251352/jom-1.1.4.zip)

@NancyLi1013  close or let it be open for discussion?

**Comment by NancyLi1013** [comment]
@Voskrese 

Thanks for your update and info.
If the new version for `jom `is released, please feel free to reopen this or open a new issue.

**Comment by Voskrese** [comment]
https://wiki.qt.io/Jom

> Getting the Source
> 
> The source code repository is hosted at codereview.qt-project.org and mirrored on code.qt.io.
> 
> git clone git://code.qt.io/qt-labs/jom.git

**Comment by xavier2k6** [comment]
There's a new version of jom available at
https://download.qt.io/official_releases/jom/jom_1_1_4.zip

Changes since jom 1.1.3
- Fixed handling of the command line option /j.
- Makefiles that do not contain targets are allowed.
- Fixed handling of macro definitions on the command line
  (QTCREATORBUG-22176).
- Enabled usage of paths longer than MAX_PATH (QTCREATORBUG-22848).
- Added version information to the executable (QTCREATORBUG-23290).
- Fixed directory separators in the MAKEDIR variable
  (QTCREATORBUG-23559).
- Fixed expansion of absolute path wildcard dependencies
  (QTCREATORBUG-26745).
- Fixed performance penalty when checking for dependency cycles in
  complex Makefiles (QTCREATORBUG-29412).

**Comment by dg0yt** [comment]
@xavier2k6 https://github.com/microsoft/vcpkg/pull/34090.
